edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/agentStart.sh

```diff
@@ -85,6 +85,8 @@ then
     chmod 600 "$backupjson"
   fi
 
+  echo "$(date --utc +"%Y-%m-%d %H:%M:%S %:z") Completed necessary setup. Starting EdgeAgent."
```

Comment:
Is this something we need to check in? If so, it should say "Edge Agent" (with a space), similar to the echo at the [top](https://github.com/Azure/iotedge/blob/2deeffaf3560ed6b6c0cd7afbcca7281fbca94e7/edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/agentStart.sh#L26) of the script.

Reply:
I wanted to check it in because I felt it was confusing for me when an error occurred without this log. I didn't know if the error was coming from a setup step or the dotnet command. Will fix.